software/nosql/pom.xml

```diff
@@ -295,4 +295,26 @@
         </plugins>
       </pluginManagement>
     </build>
+
+  <profiles>
+    <profile>
+      <id>defaultProfile</id>
+      <activation>
+	<property><name>!brooklyn.cloudtest</name></property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <excludes>
+                <exclude>**/CassandraFabricTest.java</exclude>
```

Comment:
`CassandraFabricTest` is designed to execute on localhost. It starts an `EmptySoftwareProcess` instead of real instances, so no commands get executed in reality. If the test is failing there is some problem within the test. It passes for me so probably is timing sensitive.
If the test is disabled it will go into oblivion and rot, that's why I think it's better to leave it enabled on master so it is fixed eventually, and disable it just for the release.